x-pack/plugins/ml/server/routes/README.md

```diff
@@ -6,11 +6,15 @@ Each route handler requires [apiDoc](https://github.com/apidoc/apidoc) annotatio
 to generate documentation.
 The [apidoc-markdown](https://github.com/rigwild/apidoc-markdown) package is also required in order to generate the markdown.
 
-For now the process is pretty manual. You need to make sure the packages mentioned above are installed globally 
+There are custom parser and worker (`x-pack/plugins/ml/server/routes/apidoc_scripts`) to process api schemas for each documentation entry. It's written with typescript so make sure all the scripts in the folder are compiled before executing `apidoc` command.
+
+For now the process is pretty manual. You need to make sure the latest versions of the packages mentioned above are installed globally 
```

Comment:
You should add a sentence to say you need the `typescript` package installed, as well as needing `apiDoc` and `apidoc-markdown`.

Reply:
I've introduced a `package.json` for ml plugin, so there is no need to have global dependencies anymore